### PR TITLE
adding descriptions

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/7188

descriptions taken from Set-

### CODE REVIEW

**Comment by get-itips** [comment]
@yogkumgit

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -893,7 +894,12 @@ Accept wildcard characters: False
 ```
 
 ### -WhoCanRegister
-This parameter is reserved for internal Microsoft use.
+Controls the attendees who can attend a webinar meeting. The default is Everyone, meaning that everyone can register. If you want to restrict registration  to internal accounts set the value  to 'EveryoneInCompany'.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:897]:
```suggestion
Controls the attendees who can attend a webinar meeting. The default is Everyone, meaning that everyone can register. If you want to restrict registration to internal accounts, set the value to 'EveryoneInCompany'. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@get-itips copy review ( as suggested by Johan)

**Review by dariomws — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**